### ISSUE TITLE
SCF maintenance and support

### ISSUE THREAD

**Issue by rhexor** [WordPress/secure-custom-fields#19]
Since the SCF support forum is currently not accepting new registrations, the only available option to post questions about SCF appears to be GitHub. Assuming yes, what's the best way to confirm if SCF will continue to be maintained and supported given the recent changes with the SCF plugin update reverting back to ACF? The background to the question is that if users would like to continue to use SCF and not ACF, is SCF being actively maintained and supported plus will this continue into the future?

**Comment by kraftbj** [comment]
Howdy! It does seem that new user registrations on wordpress.org are paused right now, but happy to answer on GitHub best I can.

Yes, the plan is to continue to maintain and evolve SCF going further. While I'm not a fan of how we got to this point, custom fields are an important part of WordPress as a CMS and an application framework. While I'm about to log off to try to unplug during the holidays, I'd like to integrate SCF into the work of the broader WP contributor community.

In short, SCF will continued to be maintained.

**Comment by kraftbj** [comment]
Closing, but we can continue to discuss.
